src/main/java/com/foo/gosucatcher/domain/member/presentation/MemberController.java

```diff
@@ -39,52 +46,84 @@ public class MemberController {
+		//todo: 이메일 인증 시스템 만들기
+		MemberPasswordFoundResponse response = memberService.findPassword(email);
 
-		return ResponseEntity.ok(null);
+		return ResponseEntity.ok(response);
 	}
 
-	@PostMapping("/login")
-	public ResponseEntity<MemberLogInResponse> logIn(
-		@RequestBody @Validated MemberLogInRequest memberLogInRequest) {
-		MemberLogInResponse response = memberService.logIn(memberLogInRequest);
+	@GetMapping("/signup")
+	public ResponseEntity<MemberEmailDuplicateResponse> checkDuplicatedEmail(
+		@RequestParam @Validated @Email(message = "올바른 이메일 형식을 입력하세요") String email) {
+		MemberEmailDuplicateResponse response = memberService.checkDuplicatedEmail(email);
 
 		return ResponseEntity.ok(response);
 	}
 
-	@GetMapping("/signup")
-	public ResponseEntity<Void> checkDuplicatedEmail(@RequestParam @Validated @Email String email) {
-		memberService.checkDuplicatedEmail(email);
+	@PostMapping("/signup")
+	public ResponseEntity<MemberSignupResponse> signup(
+		@RequestBody @Validated MemberSignupRequest memberSignUpRequest) {
+		MemberSignupResponse response = memberService.signup(memberSignUpRequest);
 
-		return ResponseEntity.ok(null);
+		return ResponseEntity.status(HttpStatus.CREATED)
+			.body(response);
 	}
 
-	@GetMapping("/{email}")
-	public ResponseEntity<MemberPasswordFoundResponse> findPassword(@PathVariable @Validated @Email String email) {
-		MemberPasswordFoundResponse response = memberService.findPassword(email);
+	@PostMapping("/login")
+	public ResponseEntity<MemberCertifiedResponse> login(
+		@RequestBody @Validated MemberLoginRequest memberLoginRequest) {
+		MemberCertifiedResponse response = memberService.login(memberLoginRequest);
 
 		return ResponseEntity.ok(response);
 	}
 
-	@PatchMapping("/{memberId}")
-	public ResponseEntity<Long> changeMemberInfo(@PathVariable long memberId,
-		@RequestBody @Validated MemberInfoChangeRequest memberInfoChangeRequest) {
-		long responseId = memberService.changeMemberInfo(memberId, memberInfoChangeRequest);
+	@PostMapping("/refresh-token")
+	public ResponseEntity<MemberCertifiedResponse> refreshToken(
+		@RequestBody @Validated MemberRefreshRequest memberRefreshRequest) {
+		MemberCertifiedResponse response = memberService.refresh(memberRefreshRequest);
 
-		return ResponseEntity.ok(responseId);
+		return ResponseEntity.ok(response);
+	}
+
+	@CurrentMemberEmail
+	@DeleteMapping("/logout")
+	public ResponseEntity<Void> logout(String memberEmail) {
+		memberService.logout(memberEmail);
+
+		return ResponseEntity.noContent().build();
 	}
 
-	@DeleteMapping("/{memberId}")
-	public ResponseEntity<Void> deleteMember(@PathVariable long memberId) {
+	@CurrentMemberId
+	@DeleteMapping("/me")
+	public ResponseEntity<Void> deleteMember(Long memberId) {
 		memberService.deleteMember(memberId);
 
-		return ResponseEntity.ok(null);
+		return ResponseEntity.noContent().build();
+	}
+
+	@CurrentMemberId
+	@GetMapping("/me/profile")
```

Comment:
URI자원들은 어색한 경우가 아니라면 복수형으로 해주는게 rest api에 더 맞는거 같아요
```suggestion
	@GetMapping("/me/profiles")
```

Reply:
반영하겠습니다~!